src/irmin-pack/s.ml

```diff
@@ -79,6 +79,12 @@ module type S = sig
     ([> `Msg of string ], [> `Msg of string ]) result Lwt.t
 
   val reconstruct_index : ?output:string -> Irmin.config -> unit
+
+  val traversal :
```

Comment:
Something like `stats` instead of `traversal`?

Reply:
I renamed it in `Stats.run`, I can change it if you prefer something else